src/commands/app/doctor.ts

```diff
@@ -0,0 +1,240 @@
+    const anonymousNotRestricted = _.isEqual(anonymous.controllers, {
+      "*": { actions: { "*": true } },
+    });
+
+    if (adminExists) {
+      this.logOk(`An admin user exists`);
+    } else {
+      this.logKo("No admin user exists");
+      suggestions.push(
+        `Create an admin user ${
+          anonymousNotRestricted ? "and restrict anonymous role " : ""
+        }with
+           kourou security:createFirstAdmin '{
+              credentials: {
+                local: {
+                  username: "admin",
+                  password: "password"
+                }
+              }
+            }' ${anonymousNotRestricted ? "-a reset=true" : ""}`
+      );
+    }
+
+    if (anonymousNotRestricted && adminExists) {
+      this.logKo(`Anonymous role does not restrict access to any controller`);
+      suggestions.push(
+        `Restrict anonymous role controllers with
+           kourou security:updateRole '{
+            "controllers": {
+              "auth": {
+                "actions": {
+                  "checkToken": true,
+                  "getCurrentUser": true,
+                  "getMyRights": true,
+                  "login": true
+                }
+              },
+              "server": {
+                "actions": {
+                  "publicApi": true,
+                  "openapi": true
+                }
+              }
+            }
+          }' --id=anonymous
+          // or make your own restrictions (see: https://docs.kuzzle.io/core/2/guides/main-concepts/permissions/#roles)`
+      );
+    }
+    const config = await this.sdk.server.getConfig({});
+    if (config) {
+      if (config.cluster.enabled) {
+        this.logOk(`Cluster is enabled`);
+      } else {
+        this.logKo("Cluster is disabled");
+      }
+    } else {
+      this.logInfo("No config found for the server");
+    }
+
+    // Redis
+    if ("PONG" === (await this.sdk.ms.ping())) {
+      this.logOk("Redis Memory Storage is running");
+    } else {
+      this.logKo("=> Redis Memory Storage is not running");
+      suggestions.push("Check Redis instance dedicated to memory storage");
+    }
+
+    this.logOk("Kuzzle Version: " + config.version);
+    this.logOk("Kuzzle NodeJS version: " + nodeVersion.result);
+    if (nodeVersion.result > this.NODEJS_MAX_VERSION) {
+      this.logKo(
+        "=> Kuzzle NodeJS version is not compatible with Kuzzle (max version is " +
+          this.NODEJS_MAX_VERSION +
+          ")"
+      );
+    }
+    this.logOk("NodeJS Building version: " + process.version);
+    if (nodeVersion.result != process.version) {
+      this.logKo(
+        "=> Kuzzle node version is different from the build node version"
+      );
+      suggestions.push(
+        `Match Kuzzle NodeJS version ${nodeVersion.result} with the build NodeJS version ${process.version}`
+      );
+    }
+
+    let elkVersion;
+    try {
+      const client = new Client({ node: this.flags.elasticsearch });
```

Comment:
ES version should rather be fetch from `server:info` because it's quite rare to expose ES directly apart when developing locally.

Or maybe this whole part should be optionnal

Reply:
I fetch es version from server:info and try to fetch other data from ES with the client. If not possible, I throw an error.